library/Icingadb/View/HostgroupRenderer.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/* Icinga DB Web | (c) 2025 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Icingadb\View;
+
+use Icinga\Module\Icingadb\Common\Links;
+use Icinga\Module\Icingadb\Model\Hostgroupsummary;
+use Icinga\Module\Icingadb\Widget\Detail\HostStatistics;
+use Icinga\Module\Icingadb\Widget\Detail\ServiceStatistics;
+use ipl\Html\Attributes;
+use ipl\Html\HtmlDocument;
+use ipl\Html\HtmlElement;
+use ipl\Html\Text;
+use ipl\I18n\Translation;
+use ipl\Stdlib\BaseFilter;
+use ipl\Stdlib\Filter;
+use ipl\Web\Widget\ItemTable\ItemTableRenderer;
+use ipl\Web\Widget\Link;
+
+/** @implements ItemTableRenderer<Hostgroupsummary> */
+class HostgroupRenderer implements ItemTableRenderer
+{
+    use Translation;
+    use BaseFilter;
+
+    public function assembleAttributes($item, Attributes $attributes, string $layout): void
+    {
+        $attributes->get('class')->addValue('hostgroup');
+    }
+
+    public function assembleVisual($item, HtmlDocument $visual, string $layout): void
+    {
+    }
+
+    public function assembleTitle($item, HtmlDocument $title, string $layout): void
+    {
+        if ($layout === 'header') {
+            $title->addHtml(new HtmlElement(
+                'span',
+                Attributes::create(['class' => 'subject']),
+                Text::create($item->display_name)
+            ));
+        } else {
+            $link = new Link(
+                $item->display_name,
+                Links::hostgroup($item),
+                [
+                    'class' => 'subject',
+                    'title' => sprintf(
+                        $this->translate('List all hosts in the group "%s"'),
+                        $item->display_name
+                    )
+                ]
+            );
+
+            if ($this->hasBaseFilter()) {
+                $link->getUrl()->setFilter($this->getBaseFilter());
+            }
+
+            $title->addHtml($link);
+        }
+    }
+
+    public function assembleCaption($item, HtmlDocument $caption, string $layout): void
+    {
+        $caption->addHtml(new HtmlElement('span', null, Text::create($item->name)));
```

Comment:
Text doesn't need to be wrapped in a span if it has no attributes. This is the case in other renderers as well, please adjust all of them accordingly.